usr/share/lib/img_proof/tests/SLES/test_sles_multipath_off.py

```diff
@@ -1,3 +1,4 @@
 def test_sles_multipath_off(host):
-    kernel_cmdline = host.file('/proc/cmdline')
-    assert kernel_cmdline.contains(' multipath=off ')
+    kernel_cmdline = host.run('cat /proc/cmdline')
```

Comment:
Was there a reason to switch from host.file() to host.run() here? The content should be the same whether you cat or read the content of the file? The part that matters is switching to split()ing the content, and checking that one of the entries in the resulting list is the desired value...

Reply:
Yeah, since we need the entire file contents to do the string split the host.file call will end up running the cat command anyway. So this simplifies the test.

Generally I would only use the host.file wrapper if I wanted to check multiple things such as is_file, file_exists and/or checking file permissions.

Reply:
Here, I think we are safe to assume that /proc/cmdline should and if it doesn't we should error out anyway.

Reply:
Ok so host.run() is actually a subset of host.file() so yeah using host.run() makes more sense in this case, since we have bigger problems if /proc/cmdline doesn't exist of isn't a file. ;-)

Reply:
Yep, basically a subset. The contains method will just end up running cat.